src/run/docker-compose.ts

```diff
@@ -16,7 +16,7 @@ const CONTAINER_STOPPED_STATUS_PATTERN =
     /Container\s([\dA-Za-z-]*)\s*(Stopped|Removed)/;
 
 const CONTAINER_STARTED_HEALTHY_STATUS_PATTERN =
-    /Container\s([\dA-Za-z-]*)\s*(Started|Healthy|Stopped)/;
+    /(?:Container\s)?([\dA-Za-z-]*)\s*(Started|Healthy|Stopped|Pulling|Pulled)/;
```

Comment:
Is the "service name" always expected or can it be really an empty string ?
I mean, the `([\dA-Za-z-]*)` can match an empty string.
if instead that should hold some value it could be:
`([\dA-Za-z-]+)` same story for the following space: `\s*` vs `\s+`

Reply:
It is expected - I see what you mean it should be + rather than *. I'll fix